README.md

```diff
@@ -172,7 +173,7 @@ In order to use the API within `polkadot.js` you'll need to configure the follow
     "num_outputs": "u32"
   },
   "FixedMetadataValueRestriction": {
```

Comment:
ok to reuse this for two different restrictions?

Reply:
yep that's fine